React-Projects/job-portal/src/PostJob.jsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import "./styles/PostJob.css";
+import axios from "axios";
+
+const PostJob = () => {
+  let isLoggedIn = false;
+  const navigate = useNavigate();
+
+  const jwtCookie = document.cookie
+    .split(";")
+    .find((cookie) => cookie.trim().startsWith("jwt_token="));
+
+  if (jwtCookie) {
+    isLoggedIn = true;
+  }
+
+  const [jobTitle, setJobTitle] = useState("");
+  const [jobType, setJobType] = useState("");
+  const [jobSalary, setJobSalary] = useState(0);
+  const [jobDescription, setJobDescription] = useState("");
```

Comment:
@codal-kcolaco When we connect next time, let me know I will explain how to optimise this. for now you can add this.

Reply:
@abhi-codal Understood